x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/data_stream/140_data_stream_aliases.yml

```diff
@@ -193,3 +193,70 @@
   - do:
       indices.get_alias: {}
   - match: {log-foobar.aliases.my-alias: {}}
+
+---
+"Fix IndexNotFoundException error when handling remove alias action":
```

Comment:
Error occurs only with security enabled. The wildcard extension what security is doing in combination with how indices aliases api worked without this fix caused the bug. These yaml tests are ran with security enabled, data streams qa modules are currently ran with security disabled.